SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -325,7 +325,9 @@ Quarks {
 			if(fetch, { this.prFetchDirectory });
 			this.prReadDirectoryFile(dirTxtPath);
 		}.try({ arg err;
-			("Failed to read quarks directory listing: % %".format(if(fetch, directoryUrl, dirTxtPath), err)).error;
+			var text = err.tryPerform(\errorString);
+			if(text.isNil) { text = err.asString };
```

Comment:
any reason not to condense this to `err.tryPerform(\errorString) ?? { err.asString }`?

Reply:
Pending discussion below.

Reply:
Addressed.